torch/testing/_internal/hop_db.py

```diff
@@ -63,6 +63,7 @@ def f2(x, y0, y1):
     "triton_kernel_wrapper_mutation",
     "triton_kernel_wrapper_functional",
     "hints_wrapper",
+    "invoke_subgraph",
```

Comment:
I'd recommend adding it to the HOP db and looking at the tests. All the HOPs in the HOP db go through strenuous autogenerated tests

Reply:
Added.